Makefile

```diff
@@ -24,7 +24,7 @@ clean-build:
 	rm -fr .eggs/
 	rm -fr dist-packages/
 	rm -fr dist-packages-temp/
-	rm -f kalite/database/data.sqlite
```

Comment:
...which is why it gets deleted explicitly in make dist and make release targets!